resources/lang/en/thoughts/alive/general.json

```
@@ -5494,6 +5494,276 @@
             "living"
         ]
     },
+    {
+        "id": "allergies_gen",
+        "thoughts": [
+            "Is having a sneezing fit",
+            "Is having an extra sniffily day",
+            "Is having a non-sniffily day",
+            "Sounds like {PRONOUN/m_c/subject}{VERB/m_c/'re/'s} hacking up a hairball",
+            "Is laughing about sounding like an elder",
```

```suggestion
            "Laughs about sounding like an elder",
```
'ing' words one right after another end up sounding a little awkward